### DISCUSSION TITLE
INFO: Could not find files for the given pattern(s).

### DISCUSSION THREAD

**Discussion by thedevildude** [smartcontractkit/full-blockchain-solidity-course-py#47] [Q&A] [answered]
Everything works fine so far and I don't get any compilation error but I get this line in my console every time I run **Brownie** or **python deploy.py**.
Is it normal?

**Comment by AlbertoSinigaglia** [comment]
...which line?

**Reply by thedevildude** [reply]
**INFO: Could not find files for the given pattern(s)**

I get this on console when I run the scripts, however the scripts run fine without any error

**Reply by AlbertoSinigaglia** [reply]
Are you sure you have your `.env` file created? Because if you add 
```
dotenv: .env
```
In your config but you don't create the file, you will get this error

**Reply by thedevildude** [reply]
> Are you sure you have your `.env` file created? Because if you add
> 
> ```
> dotenv: .env
> ```
> 
> In your config but you don't create the file, you will get this error

Yes I have created the .env file and perfectly able to extract variables from it

**Comment by PatrickAlphaC** [comment, marked as answer]
You can safely ignore this. It's not an error. `INFO` just means it's giving you some information. 

https://stackoverflow.com/questions/68412439/info-could-not-find-files-for-the-given-patterns-error
